uber/reports.py

```diff
@@ -0,0 +1,48 @@
+from uber.common import *
+
+
+class ReportBase:
+    def write_row(self, row, out):
+        # plugins can override this hook
+        out.writerow(row)
+
+
+class PersonalizedBadgeReport(ReportBase):
+    """
+    Generate a CSV file which contains personalized badges with custom printed_names on them
+    """
+
+    def __init__(self, include_badge_nums=True):
+        self._include_badge_nums = include_badge_nums
+
+    def run(self, out, session, *filters, order_by=None, badge_type_override=None):
```

Comment:
Shouldn't `*filters` be at the end? I'm not sure if it's just traditional or if there's a real reason it's always done that way.